examples/tpu/tpuvm_mnist.yaml

```diff
@@ -14,10 +14,11 @@ setup: |
     conda create -n flax python=3.10 -y
     conda activate flax
     # Make sure to install TPU related packages in a conda env to avoid package conflicts.
-    pip install "jax[tpu]==0.4.26" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
-    pip install clu
+    pip install \
+      -f https://storage.googleapis.com/jax-releases/libtpu_releases.html "jax[tpu]==0.4.25" \
+      clu \
+      tensorflow tensorflow-datasets
     pip install -e flax
-    pip install tensorflow tensorflow-datasets
```

Comment:
nit: Is there any specific reason to change the format here? Can we just change `jax[tpu]==0.4.26` to `jax[tpu]==0.4.25`?

Reply:
Yes because if we only change the version of `jax`, it will be updated then since the newest version of `clu` requires `jax>=0.4.26`. That's why I put them together. Another solution is to specify the version of `clu`.